email_account_validity/_config.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Optional
+
+import attr
+
+
+@attr.s
+class EmailAccountValidityConfig:
+    period = attr.ib(type=int)
+    renew_at = attr.ib(type=int)
+    renew_email_subject = attr.ib(type=Optional[str], default=None)
```

Comment:
for future reference, `auto_attribs` is nice.

```suggestion
@attr.s(auto_attribs=True)
class EmailAccountValidityConfig:
    period: int
    renew_at: int
    renew_email_subject: Optional[str] = None
```

Reply:
Oh I thought it only worked for attribs that had a default value, TIL